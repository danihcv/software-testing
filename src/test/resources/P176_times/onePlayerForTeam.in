7 7
a 12
b 5
c 62
d 2
e 2
f 58
g 7
