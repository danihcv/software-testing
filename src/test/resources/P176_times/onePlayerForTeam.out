Time 1
c

Time 2
f

Time 3
a

Time 4
g

Time 5
b

Time 6
e

Time 7
d
